### PR TITLE
Course improvements

### PR DESCRIPTION
**Pull request by fabianaferreira**
Fixing typos

### CODE REVIEW

File: course/steps/04-events-triggers/en.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Events: Using Events as Triggers
+# Using events as triggers
```
Comment (juliarocha) [inline course/steps/04-events-triggers/en.md]: Changing this title requires changes in the  `config.json`, right? And some steps are in the previous pattern as well, e.g.: Clients: Using Master Data, do you think it makes sense to change all of them?

**Review by juliarocha — CHANGES_REQUESTED** [review]
Looks nice :noice: just that one comment on the steps' titles 🚀

File: course/steps/04-events-triggers/en.md (hunk 1 above)
Comment (fabianaferreira) [reply course/steps/04-events-triggers/en.md]: I'll just forget about changing it :hehehe:

**Review by juliarocha — APPROVED** [review]
Go for it 🚀

**Pull request merged**